### DISCUSSION TITLE
Debug mode?

### DISCUSSION THREAD

**Discussion by henrikbossart** [amannn/next-intl#1254] [Q&A]
Hi

I couldn't find anything about this in the docs, but I was wondering if there is some sort of debug mode, that returns the keys/'paths' of the message values instead of the actual value?

**Comment by amannn** [comment]
Hey! There's currently no such mode, no.

However, you can pass an empty object `{}` to `messages`, this will effectively result in placeholders being rendered everywhere. 

**Example:**

![Screenshot 2024-08-16 at 13 33 08](https://github.com/user-attachments/assets/c3555383-a786-4d17-bde1-c887b0b6bce4)


You might want to consider configuring [error handling](https://next-intl-docs.vercel.app/docs/usage/configuration#error-handling) if you use this.
